### PR TITLE
updated structure and added new examples

### PR DESCRIPTION
**Pull request by NicholasTancredi**

### CODE REVIEW

File: .pylintrc [hunk 1]
```diff
@@ -0,0 +1,301 @@
+Skip to content
```
Comment (willbenmitch) [inline .pylintrc]: Was this added manually, or autogenerated by some pylint command? just curious

File: setup.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+setup(
+    name='Python Examples',
+    version='0.0.1',
+    description='Python example code',
+    url='https://github.com/w0251251/python_examples',
+    author='Nicholas Tancredi',
+    author_email='nicholastancredi@gmail.com',
+    license='GPL3',
+    packages=find_packages(),
+    ext_modules=cythonize("examples/cython_array_sum/index.pyx"),
```
Comment (willbenmitch) [inline setup.py:14]: any documentation around what this does?

Comment (NicholasTancredi) [reply setup.py:14]: https://cython.readthedocs.io/en/latest/src/tutorial/cython_tutorial.html

File: .pylintrc (hunk 1 above)
Comment (NicholasTancredi) [reply .pylintrc]: Fixed this obviously wrong `.pylintrc` file

**Pull request merged**